tests/pipeline_with_param_table_test.py

```diff
@@ -719,3 +719,23 @@ def to_str(x: int) -> str:
     pl = sl.Pipeline((to_str,))
     pl.set_param_series(int, ints)
     assert pl.compute(sl.Series[int, str]) == sl.Series(int, {1: '1', 2: '2', 3: '3'})
+
+
+def test_multiple_param_series_can_be_broadcast() -> None:
```

Comment:
Did you also try putting the beam center finder provider back into the loki direct beam workflow with multiple files to see if it fixes it?

Reply:
I can also try it if you prefer.

Reply:
I did not try, no. Go ahead, since you know which pieces to touch!